go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kopia/kopia
 
-go 1.17
+go 1.19
```

Comment:
We may have already done this change (LM check) or may want to do it separately.